src/initializeInpageProvider.ts

```diff
@@ -29,6 +33,10 @@ type InitializeProviderOptions = {
    * Whether the window.web3 shim should be set.
    */
   shouldShimWeb3?: boolean;
+  /**
+   * The name of the stream used to connect to the wallet.
+   */
+  jsonRpcStreamName?: string | undefined;
```

Comment:
```suggestion
  jsonRpcStreamName?: string;
```

is the undefined needed?

Reply:
I believe it shouldn't the parameter is already optional but this was existing I just moved the property out of InpageProvider
I just removed it